### PR TITLE
Always localize "Password:"

### PR DESCRIPTION
**Pull request by tsujan**
Closes https://github.com/lxqt/lxqt-policykit/issues/146

NOTE: I also updated the translations and edited them to keep them complete.

### CODE REVIEW

File: translations/lxqt-policykit-agent_ar.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!DOCTYPE TS>
-<TS version="2.1" language="ar">
+<TS version="2.1" language="ar_EG">
```
[the remark this answers is not in the document]
Comment (tsujan) [reply translations/lxqt-policykit-agent_ar.ts:3]: Qt6 does it, through `lxqt2-transupdate`.

File: translations/lxqt-policykit-agent_arn.ts [hunk 2]
```diff
@@ -30,6 +30,14 @@
         <translation type="unfinished"></translation>
     </message>
 </context>
+<context>
```
[the remark this answers is not in the document]
Comment (tsujan) [reply translations/lxqt-policykit-agent_arn.ts:33]: Translations are always like this: source and ui are separate.

[the remark this answers is not in the document]
Comment (palinek) [reply translations/lxqt-policykit-agent_arn.ts:33]: > Translations are always like this: source and ui are separate.

It doesn't need to be such.

[the remark this answers is not in the document]
Comment (palinek) [reply translations/lxqt-policykit-agent_arn.ts]: Please, review/test my commit d087cd5

[the remark this answers is not in the document]
Comment (tsujan) [reply translations/lxqt-policykit-agent_arn.ts]: Very nice! It should work. I'll test it and remove all changes in translations if it works fine.

[the remark this answers is not in the document]
Comment (palinek) [reply translations/lxqt-policykit-agent_arn.ts]: > and remove all changes in translations

It is already done in the commit.

[the remark this answers is not in the document]
Comment (tsujan) [reply translations/lxqt-policykit-agent_arn.ts]: I meant no translation in the PR, as we do in usual circumstances.

**Comment by tsujan** [comment]
Tested and it worked. I think it can be merged now; can't it?

**Comment by palinek** [comment]
I wouldn't remove the new reference in .ts files done in a4d640b, as next `lupdate` will readd them anyways.

**Comment by tsujan** [comment]
The division of work: @stefonarch updates translation and does all other jobs about them. We also recommend that code contributors don't touch translations.

**Comment by tsujan** [comment]
Tested again without installation, and it worked as intended with the unmodified translations. Merging...

File: translations/lxqt-policykit-agent_ar.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (stefonarch) [reply translations/lxqt-policykit-agent_ar.ts]: > Qt6 does it, through `lxqt2-transupdate`.

I never experienced this, if you look at all the `*.ts files` I updated for wayland, it's often just `de` or `nl`.,

[the remark this answers is not in the document]
Comment (tsujan) [reply translations/lxqt-policykit-agent_ar.ts]: Perhaps it happens only when a string is added. With the change @palinek made to the code, no string is added, but it's good to update translations before the release (which you always do).

File: translations/lxqt-policykit-agent_arn.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (palinek) [reply translations/lxqt-policykit-agent_arn.ts]: > I meant no translation in the PR

You updated the translations here at the beginning.

**Pull request merged**